platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -5105,6 +5109,7 @@ DisplayServerX11::WindowID DisplayServerX11::_create_window(WindowMode p_mode, V
 			if (wt_atom != None && type_atom != None) {
 				XChangeProperty(x11_display, wd.x11_window, wt_atom, XA_ATOM, 32, PropModeReplace, (unsigned char *)&type_atom, 1);
 			}
+			window_set_ime_active(true, id);
```

Comment:
While my tests showed no obvious regressions, I am not entirely sure, if this is the right place and the right way to activate the Window during creation.

Reply:
This will break keyboard input if IME keyboard layout is used, IME should be activated only when input field (TextEdit, LineEdit) is focused, and inactive in all other cases.